rootfs/rootfs/etc/rc.d/automount

```diff
@@ -34,6 +34,37 @@ if [ ! -n "$BOOT2DOCKER_DATA" ]; then
             mkfs.ext4 -L $LABEL $BOOT2DOCKER_DATA
             swapon "${UNPARTITIONED_HD}2"
         fi
+
+        DISK_VENDOR=`cat /sys/class/block/$(basename $UNPARTITIONED_HD /dev/)/device/vendor /sys/class/block/$(basename $UNPARTITIONED_HD /dev/)/device/model | tr -d "\n"`
+        # Test if disk is "VMware, VMware Virtual S" and empty.
+        if [ "$DISK_VENDOR" = "VMware, VMware Virtual S" ]; then
+            # Check whether the disk has any known partitions on it
+            blkid -o device $UNPARTITIONED_HD
+            if [ $? == 2 ]; then
+                # As there are no partitions, let's make sure the disk is empty for real
+                dd if=$UNPARTITIONED_HD of=device_test_file bs=1k count=256 > /dev/null 2>&1
+                NON_NUL=$(<device_test_file tr -d '\0\n' | wc -c)
```

Comment:
Nice -- love that you're going the extra mile to verify for sure that the disk is definitely zeroed out (at least for the first 256k).